.github/workflows/test-build.yml

```diff
@@ -1,19 +1,18 @@
 name: Test build
+
 on:
   pull_request:
     branches:
       - master
-defaults:
-  run:
-    shell: bash
+
 jobs:
   build:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-latest
```

Comment:
Personal preference, never use latest, it will break one day.

Reply:
I think that's unlikely, but I'll revert back to `ubuntu-24.04` just to be safe. Thanks!